dplace_app/static/partials/societies/tree.html

```diff
@@ -58,7 +58,9 @@
           </ul>
     </div>
 
-    <p ng-show="results.selectedTree" style="width:500px">Note: trees have been pruned to display only societies present in D-PLACE.</p>
+    <p ng-show="results.selectedTree" style="width:500px">
+        Note: trees have been pruned to display only societies present in D-PLACE.</p>
+        <blockquote ng-show="results.selectedTree.source">{{ results.selectedTree.source.reference }}</blockquote>
```

Comment:
@Bibiko this seems to be the correct way to display source info for a tree, right?